src/js/components/DataFilter/__tests__/DataFilter-test.tsx

```diff
@@ -271,6 +273,30 @@ describe('DataFilter', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('range inclusive', () => {
```

Comment:
can we make a slightly more descriptive test name? like "includes range min and max in filtered results"

Reply:
Yes, done in latest commit